pygmt/modules.py

```diff
@@ -11,14 +11,16 @@
     data_kind,
     dummy_context,
     fmt_docstring,
+    kwargs_to_strings,
     use_alias,
 )
 
 
 @fmt_docstring
-@use_alias(V="verbose")
+@use_alias(C="tab", F="geographic", L="zdata", M="zextemes", R="region", V="verbose")
```

Comment:
Aliases should ideally follow [`pygmt.info`](https://www.pygmt.org/v0.2.1/api/generated/pygmt.info.html) for `C`, `I` and `T`, either we change `pygmt.grdinfo` (here) or `pygmt.info` (breaking change).

- C = per_column
- I = spacing (add this)
- T = nearest_multiple (add this)

Also, GMT.jl uses these aliases, which we should use if possible (but discuss first):

- D = tiles
- L = force_scan
- M = minmax_pos

Reply:
I'm fine with keeping this in line with the Julia formatting. I'll commit it now and we can discuss.